src/frontend/JasmineGraphFrontEnd.cpp

```diff
@@ -1195,23 +1193,16 @@ void *frontendservicesesion(std::string masterIP, int connFd, SQLiteDBInterface
                 continue;
             }
 
-            std::string federatedEnabled = Utils::getJasmineGraphProperty("org.jasminegraph.federated.enabled");
-
-            if (federatedEnabled == "true") {
-                JasmineGraphServer *jasmineServer = new JasmineGraphServer();
-                if (Utils::getJasmineGraphProperty("org.jasminegraph.fl.org.training") == "true") {
-                    frontend_logger.log("Initiate org communication", "info");
-                    jasmineServer->initiateOrgCommunication(graphID, trainData, sqlite);
-
-                } else {
-                    frontend_logger.log("Initiate communication", "info");
-                    jasmineServer->initiateCommunication(graphID, trainData, sqlite);
-                }
+            JasmineGraphServer *jasmineServer = new JasmineGraphServer();
+            if (utils.getJasmineGraphProperty("org.jasminegraph.fl.org.training") == "true") {
```

Comment:
```suggestion
            if (Utils::getJasmineGraphProperty("org.jasminegraph.fl.org.training") == "true") {
```

This is a static method.

Reply:
Ok, I fixed that in https://github.com/miyurud/jasminegraph/pull/201/commits/ef51224ec5ab0f48167bbbfbb0902fb7af62a5a4